src/app/slash_plugins/leaderboard.plugin.ts

```diff
@@ -0,0 +1,103 @@
+import { CommandInteraction, User } from 'discord.js';
+import { Command } from '../../common/slash';
+import { IContainer } from '../../common/types';
+import { GameType } from '../../services/gameleaderboard.service';
+
+const gameTypes = ['TicTacToe', 'ConnectFour'];
```

Comment:
We should probably make this an enum instead of an array to avoid the gymanstics of getting the index and using that